ukat/data/tests/test_fetch.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
+import os
 import pytest
 
+from dipy.data.fetcher import _get_file_md5
```

Comment:
Is this being used anywhere in ukat?

Reply:
No, I must have been using it when I was drafting some tests and removed it later.